src/main/java/org/alliancegenome/curation_api/constants/VocabularyConstants.java

```diff
@@ -69,6 +69,11 @@
 
 	public static final String GENE_EXPRESSION_VOCABULARY = "gene_expression";
 	public static final String GENE_EXPRESSION_RELATION_TERM = "is_expressed_in";
+	
+	public static final String TRANSCRIPT_RELATION_VOCABULARY = "transcript_relation";
+	public static final String TRANSCRIPT_CHILD_TERM= "is_child_of";
+	public static final String TRANSCRIPT_PARENT_TERM= "is_parent_of";
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'=' is not preceded with whitespace.
<!-- __reviewdog__:ChA0YjdmN2MyNmE1MGU5YjU5EgpjaGVja3N0eWxl -->